app/src/main/java/br/com/catbag/gifreduxsample/asyncs/net/downloader/FileDownloader.java

```diff
@@ -31,9 +28,7 @@
     private static DownloadRoute sRoutes;
 
     public FileDownloader() {
-        if (sRoutes == null) {
-            sRoutes = new RoutesGenerator().createRoutes(DownloadRoute.class);
-        }
+        sRoutes = RetrofitBuilder.getInstance().createApiEndpoint(DownloadRoute.class, RiffsyRoutes.BASE_URL);
```

Comment:
Maybe pass BASE_URL on constructor make FileDownloader more host independent, i dont know....this Retrofit endpoint url dependency its strange when it is used to file download.